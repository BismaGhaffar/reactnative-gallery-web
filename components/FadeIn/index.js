import styled, { keyframes } from 'styled-components'
import { fadeIn } from 'react-animations'

const fader = keyframes`${fadeIn}`

const FadeIn = styled.div`
  animation: ${props => props.timer || 1}s ${props => props.delay || 0}ms
    ${fader};
`

export default FadeIn
