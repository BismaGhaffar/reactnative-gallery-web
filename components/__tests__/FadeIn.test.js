import React from 'react'
import renderer from 'react-test-renderer'
import FadeIn from '../FadeIn'

it('FadeIn can be created', () => {
  const comp = renderer.create(<FadeIn />)
  expect(comp).toBeDefined()
})

it('<FadeIn /> toMatchSnapshot', () => {
  const tree = renderer.create(<FadeIn />).toJSON()
  expect(tree).toMatchSnapshot()
})
