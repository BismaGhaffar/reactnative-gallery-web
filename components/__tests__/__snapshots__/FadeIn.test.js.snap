// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FadeIn /> toMatchSnapshot 1`] = `
<div
  className="FadeIn-s1muu1d1-0 jOnkYB"
/>
`;
